source/api/index.md

```diff
@@ -50,6 +50,12 @@ hexo.call('generate', {}).then(function(){
 });
 ```
 
+``` js
+hexo.call('list', { _: ['post'] }).then(function() {
+  console.log(arguments);
```

Comment:
As a reference documentation, we should document what `arguments` are, i.e.
```js
const { exec } = require('child_process');
exec('cat *.js bad_file | wc -l', (error, stdout, stderr) => {
  if (error) {
    console.error(`exec error: ${error}`);
    return;
  }
  console.log(`stdout: ${stdout}`);
  console.log(`stderr: ${stderr}`);
});
```
is much better than
```js
const { exec } = require('child_process');
exec('cat *.js bad_file | wc -l', (...args) => {
  console.log(args);
})
```

Reply:
ups, i will remove console.log, hexo.call method is not returning any result